server/app/services/telegram_bot_service.py

```
@@ -0,0 +1,442 @@
+                async with session.post(
+                    self.lokam_webhook_url, 
+                    headers=headers, 
+                    json=webhook_payload,
+                    timeout=aiohttp.ClientTimeout(total=30)
+                ) as response:
+                    response_text = await response.text()
+                    
+                    if response.status in [200, 201, 202]:
+                        try:
+                            return await response.json()
+                        except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)